include/Global.h

```diff
@@ -196,6 +196,9 @@ extern bool            GRACKLE_CMB_FLOOR;
 extern bool            GRACKLE_PE_HEATING;
 extern double          GRACKLE_PE_HEATING_RATE;
 extern char            GRACKLE_CLOUDY_TABLE[MAX_STRING];
+extern int             GRACKLE_THREE_BODY_RATE;
+extern int             GRACKLE_CIE_COOLING;
```

Comment:
`int` --> `bool` for `GRACKLE_CIE_COOLING` and `GRACKLE_H2_OPA_APPROX`

Reply:
Sure. How about set `GRACKLE_CIE_COOLING ` to `bool`, and keep `GRACKLE_H2_OPA_APPROX` in `int` since it is likely to have multiple optical approximations in different version of grackle.

Reply:
Good suggestion